health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserAccess.js

```
@@ -283,8 +286,17 @@ function UserAccess({ category, setData, nationalRoles }) {
-          onOverlayClick={() => { }}
-          footerChildren={[<Button type={"button"} size={"large"} variation={"secondary"} label={t("CLOSE")} onClick={() => setShowPopUp(false)} style={{ minWidth: "200px" }} />]}
+          onOverlayClick={() => {}}
+          footerChildren={[
+            <Button
+              type={"button"}
+              size={"large"}
+              variation={"secondary"}
+              label={t("CLOSE")}
+              onClick={() => setShowPopUp(false)}
+              style={{ minWidth: "200px" }}
+            />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)